public/app/features/dashboard/components/GenAI/GenAIButton.tsx

```diff
@@ -197,4 +202,7 @@ const getStyles = (theme: GrafanaTheme2) => ({
   wrapper: css({
     display: 'flex',
   }),
+  spinner: css({
+    color: '#6E9FFF',
```

Comment:
does this color come from theme? (sometimes figma will share this info / if it comes from a theme preset)

Reply:
oops, should've done a better job at cleaning it up, made the other PR and forgot to return to this one.. 😬 updated.